bor/README.md

```diff
@@ -0,0 +1,166 @@
+# PRELIMINARY TERMINOLOGY
+
+* A `side-transaction` is a normal heimdall transaction but the data with which the message is composed needs to be voted on by the validators since the data is obscure to the consensus protocol itself and it has no way of validating the data's correctness.
+* A `sprint` comprises of 16 bor blocks.
+* A `span` comprises of 100 sprints in bor.
```

Comment:
I think it's 400 now with 16 block sprint, Also sprint is decided by bor and not heimdall.

Reply:
Added in [this](https://github.com/maticnetwork/heimdall/pull/1071/commits/096779cf5bb71ba835d04dd5fa866515ed045a6a) commit.